common/app/views/fragments/amp/stylesheets/liveBlog.scala.html

```diff
@@ -78,11 +78,11 @@
 }
 
 .liveblog-block-byline__img {
-    display: inline;
-    border-radius: 50%;
-    max-width: 2.25rem;
     vertical-align: middle;
-    margin-right: 0.25rem;
+}
+
+.liveblog-block-byline__img img {
```

Comment:
Guessing this might be an amp thing, but should we be styling elements directly?